ramalama/model_factory.py

```diff
@@ -119,7 +127,7 @@ def create_url(self) -> URL:
         return model
 
 
-def New(name, args, transport: str = None) -> Union[Huggingface | ModelScope | Ollama | OCI | URL]:
+def New(name, args, transport: str | None = None) -> Union[Huggingface | ModelScope | Ollama | OCI | URL]:
```

Comment:
The return type could be replaced here, right?
```suggestion
def New(name, args, transport: str | None = None) -> CLASS_MODEL_TYPES:
```